.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "domain-models-runtime-it/ramls/raml-util"]
 	path = domain-models-runtime-it/ramls/raml-util
 	url = https://github.com/folio-org/raml
+[submodule "ramls/raml-util"]
+	path = ramls/raml-util
+	url = https://github.com/folio-org/raml
```

Comment:
This creates the new submodule "ramls" and creates the raml-util submodule in it.
What purpose does it serve?

Reply:
done.

Reply:
I don't know why, but build fails on: "Could not perform submodule update" both locally and on Jenkins without this

Reply:
We need to see why this is necessary.

Reply:
Using
```sh
  git submodule deinit ramls/raml-util
  git rm ramls/raml-util
```
it is gone. And RMB still compiles happily on my machine.

Reply:
@adamdickmeiss it is working on your machine, could you please commit it to the branch, so that it builds successfully on Jenkins.

Reply:
Resolved.